build.gradle

```diff
@@ -25,6 +25,19 @@ allprojects {
         google()
         jcenter()
         maven { url "https://dl.bintray.com/wordpress-mobile/maven" }
+
+        if (rootProject.ext.buildGutenbergFromSource) {
```

Comment:
Since this is now here should it be removed from `WordPress/build.gradle`?

Reply:
Good catch, makes sense! Done with bc0a199.